extensions/dependencies/guppy.html

```diff
@@ -18,6 +18,7 @@
   .guppy_active .main_cursor {
     animation: guppy-blink-animation 1s steps(2, start) infinite;
     -webkit-animation: guppy-blink-animation 1s steps(2, start) infinite;
+    color: black !important;
```

Comment:
can you use a more specific CSS selector instead of resorting to !important?

Reply:
It overrides the cursor color previously defined to black. I don't know what other selector will do it?

Reply:
See https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity (and other resources to learn about rules for CSS specificity) -- you'll need to formulate a suitable rule based on the classes that are applied to the cursor on the page. Probably `.guppy_active .main_cursor` isn't sufficient and that's why it's getting overridden.

Reply:
What I observed is-- Inline styling is in use to set the cursor color as evident from inspecting the element. So, this type of styling can only be overridden by using !important. That's why `.guppy_active .main_cursor` is getting overridden while not using !important. Thanks!

Reply:
OK, that explanation sounds fine to me. Please add a comment in the CSS to explain the rationale for the use of '!important', but otherwise this looks good. Thanks!

Reply:
Done 👍